cfme/tests/test_appliance.py

```diff
@@ -197,12 +222,14 @@ def test_certificates_present(appliance, soft_assert):
         .format(ca_cert=rhsm_ca_cert, url=rhsm_url)
     ).success
 
-    for cert in known_certs:
-        assert appliance.ssh_client.run_command("test -f '{}'".format(cert)).success
-        assert appliance.ssh_client.run_command(
-            "openssl verify -CAfile {ca_cert} '{cert_file}'"
-            .format(ca_cert=rhsm_ca_cert, cert_file=cert)
-        )
+    if appliance.version < '5.11':
+        # Until BZ(1712929) against RHEL is solved, keep this branch not ommitted on 5.11.
```

Comment:
This should be controlled by a `BZ` object and its `blocks` method, not just documented in a comment.

Reply:
@mshriver Are you sure it is going to work when the BZ is reported against RHEL?

Reply:
@mshriver it is not blocking
```
In [3]: BZ(1712929).blocks
Out[3]: False

In [6]: cfme.utils.appliance.current_appliance.version
Out[6]: Version('5.11.0.5')
```

Reply:
For now using forced streams is fine, with a comment that its RHEL related.

We _can_ update `BZ.blocks` to also resolve the OS version and BZ product, but currently it won't block on these RHEL bugs because they won't have versions set, and other markers that the `blocks` method uses to resolve a CFME blocker.